### PR TITLE
containers-and-vms/index,libvirt: Add sections.

### PR DESCRIPTION
**Pull request by flexibeast**
Also move `lxc` into `containers-and-vms`.

Partially addresses #350.

### CODE REVIEW

File: src/config/containers-and-vms/libvirt.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# libvirt
+
+[libvirt](https://en.wikipedia.org/wiki/Libvirt) is an API and daemon for
```
Comment (ericonr) [inline src/config/containers-and-vms/libvirt.md:3]: Do they have a homepage? Perhaps makes more sense to link it here.

File: src/config/containers-and-vms/index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Containers and Virtual Machines
+
+This section describes how to set up some of the container and virtual machine
+systems available on Void.
```
Comment (ericonr) [inline src/config/containers-and-vms/index.md:4]: "systems" or "tools" ?

File: src/config/containers-and-vms/libvirt.md (hunk 1 above)
Comment (flexibeast) [reply src/config/containers-and-vms/libvirt.md:3]: They do - https://libvirt.org/  - but i felt it was not particularly helpful for an introduction, at least not compared to the Wikipedia page. But yeah, you're right, it's probably more appropriate.

File: src/config/containers-and-vms/index.md (hunk 2 above)
Comment (flexibeast) [reply src/config/containers-and-vms/index.md:4]: "Tools" feels odd to me here, perhaps too specific? E.g. i'd described 'grep' as a tool, but wouldn't describing (say) LibreOffice as an "office productivity tool". How about "software"?

Comment (ericonr) [reply src/config/containers-and-vms/index.md:4]: software sounds good!

File: src/config/containers-and-vms/libvirt.md (hunk 1 above)
Comment (ericonr) [reply src/config/containers-and-vms/libvirt.md:3]: I guess linking to the homepage is better for consistency sake? If you want to prioritize the more helpful page, I don't have any issue with it. Just wanted to bring it up.

**Comment by flexibeast** [comment]
@ericonr: Thanks.
@void-linux/pkg-committers: Any comments/feedback?

File: src/config/containers-and-vms/libvirt.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+The `libvirt` package provides the [virsh(1)](https://man.voidlinux.org/virsh.1)
+interface to libvirtd. `virsh` is an interactive shell and batch-scriptable tool
+for performing management tasks, including creating, configuring and running
+virtual machines, and managing networks and storage. Note that `virsh` usually
+needs to be run as root, as described in the `virsh` man page:
+
+> Most virsh commands require root privileges to run due to the communications
+> channels used to talk to the hypervisor. Running as non root will return an
+> error.
```
Comment (ahesford) [inline src/config/containers-and-vms/libvirt.md:20]: This is true, but `libvirt` ships with a polkit rule `/usr/share/polkit-1/rules.d/50-libvirt.rules` that grants management privileges to users of the `libvirt` group. Thus, I suggest you add something like the following text:
```
As an alternative, if you have the `polkit` and `dbus` packages installed and you enable the `dbus` service, `libvirtd` will grant necessary privileges to any user added to the `libvirt` group.
```

Comment (flexibeast) [reply src/config/containers-and-vms/libvirt.md:20]: Thanks, PR updated.

**Comment by flexibeast** [comment]
@Frick-David: With this PR merged, please rebase #380 on `master`.

**Pull request merged**